tasks/db_deploy/test/test_db_deploy.py

```diff
@@ -208,11 +217,13 @@ def test_regression_test(self,
         script_filename_1 = uuid.uuid4().__str__()
         script_filename_2 = uuid.uuid4().__str__()
         drop_database = 'True'
+        prefix = uuid.uuid4().__str__()
+        os.environ['PREFIX'] = prefix
```

Comment:
Do we need to do this with the `setUp` you added?

Reply:
I prefer to have the tests manage their own test values. Theoretically we could just pull setUp's randomized value from the environment, but that would make the test reliant on setUp's implementation.

Reply:
Nevermind. Removed setUp.